conmon/cmsg.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2016 SUSE LLC
```

Comment:
The code comes from runC. If you want I can drop it (though personally I prefer having license headers in all source files).